src/routes/submission/index.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import SubmissionPage from './SubmissionPage';
+import { fetchSubmission } from '../../actions/submission';
+
+export default [
+  {
+    path: '/submission/:id',
+
+    async action({ params, store }) {
+      const submission = await store.dispatch(fetchSubmission(params.id));
```

Comment:
Not completely sure if this is the boilerplate + Redux's way to do such thing. It would be slightly simpler to just fetch it (without dispatching an action) but then we wouldn't have the object in the store.